### ISSUE TITLE
Weight initialisation in SplineCNN

### ISSUE THREAD

**Issue by mariolinovIC** [rusty1s/pytorch_geometric#1238]
## ❓ Questions & Help

Hi, 

I am interested in Message passing and in particular your SplineCNN.
I have realised that the weight initialisation in SplineCNN is relevant for the accuracy.
In your paper on SplineCNN there is no explanation on which initialisation you are using.
The one in your code uses an uniform initialisation in the range (-bound, bound) with:
`bound = 1.0 / math.sqrt(size)`
which seems to work well.

Why do use this normalisation?
Do you recommend me any reference for learning about weight initialisation?

Thank you

Mario

**Comment by rusty1s** [comment]
This initialization is just copied from `torch.nn.Linear`. In fact, we didn't play with different weight initializations at all. I am far from an expert here either, sorry!

**Comment by mariolinovIC** [comment]
Thank you for your quick reply anyway.
